lib/core/include/irods/connection_pool.hpp

```diff
@@ -15,6 +16,36 @@
 
 namespace irods
 {
+    /// Defines various options for changing the behavior of a connection pool.
+    ///
+    /// Refresh options are not mutually exclusive.
+    ///
+    /// It is important to understand that any of the options can trigger a connection
+    /// refresh. When a refresh occurs, all state related to connection refresh is reset.
+    /// Therefore, care must be taken when setting multiple options in order to obtain
+    /// deterministic refresh behavior.
+    ///
+    /// \since 4.3.1
+    struct connection_pool_options
+    {
+        /// Defines the number of times a connection can be retrieved from the pool before
+        /// refreshing it.
```

Comment:
is it 'refreshing it' or 'it refreshes'?

Reply:
or 'it is refreshed'.

Reply:
Can change to "it is refreshed".

Reply:
done